src/avram/box.cr

```diff
@@ -36,8 +36,36 @@ abstract class Avram::Box
     operation.save!
   end
 
+  # Returns an array with 2 instances of the model from the Box.
+  #
+  # Usage:
+  #
+  # ```crystal
+  # tags = TagBox.create_pair
+  # typeof(tags) # => Array(Tag)
+  # tags.size    # => 2
+  # ```
+  def self.create_pair
+    [1, 2].map do |n|
+      new.create
+    end
+  end
+
+  # Similar to `create_pair`, but accepts a block which yields the box instance.
```

Comment:
I think a newline in  here would be good so it separates the summary line from the rest of the docs